resources/templates/ha-vpn-interop-guide-template/index.md

```diff
@@ -0,0 +1,836 @@
+        - [Configure the IPsec static virtual tunnel interface (SVTI)](#configure-the-ipsec-static-virtual-tunnel-interface-svti)
+    - [Configuring the dynamic routing protocol](#configuring-the-dynamic-routing-protocol)
+    - [Saving the configuration](#saving-the-configuration)
+    - [Testing the configuration](#testing-the-configuration)
+- [Troubleshooting IPsec on \<vendor-name>\<product-name>](#troubleshooting-ipsec-on-vendor-nameproduct-name)
+- [Reference documentation](#reference-documentation)
+    - [GCP documentation](#gcp-documentation)
+    - [\<vendor-name>\<product-name> documentation](#vendor-nameproduct-name-documentation)
+
+<Put trademark statements here>: <vendor terminology> and the <vendor> logo are
+trademarks of <vendor company name> or its affiliates in the United States
+and/or other countries.
+
+_Disclaimer: This interoperability guide is intended to be informational in
+nature and shows examples only. Customers should verify this information by
+testing it._
+
+Author: <author name and email address>
+
+## Introduction
+
+This guide walks you through the process of configuring
+\<vendor-name>\<product-name> for integration with the
+[HA VPN service](https://cloud.google.com/vpn/docs) on GCP.
+
+For more information about HA or Classic VPN, see the
+[Cloud VPN Overview](https://cloud.google.com/compute/docs/vpn/overview).
+
+## Terminology
+
+Below are definitions of terms used throughout this guide.
+
+<This is some sample terminology. Add any terminology to this section that needs
+explanation.>
+
+-  **GCP VPC network**—A single virtual network within a single GCP project.
+-  **On-premises gateway**—The VPN device on the non-GCP side of the
+connection, which is usually a device in a physical data center or in
+another cloud provider's network. GCP instructions are written from the
+point of view of the GCP VPC network, so "on-premises gateway" refers to the
+gateway that's connecting _to_ GCP.
+-  **External IP address** or **GCP peer address**—a single static IP address
+within a GCP project that exists at the edge of the GCP network.
+-  **Dynamic routing**—GCP dynamic routing for VPN using the
+[Border Gateway Protocol (BGP)](https://wikipedia.org/wiki/Border_Gateway_Protocol).
+-  \<vendor-name>\<product-name> term
+-  \<vendor-name>\<product-name> term
+
+## Topology
+
+HA VPN supports [multiple topologies](https://cloud.google.com/vpn/docs/concepts/topologies):
+
+This interop guide is based on [1-peer-2-address](https://cloud.google.com/vpn/docs/concepts/topologies#1-peer-2-addresses) topology.
+
+## Product environment
+
+The \<vendor-name>\<product-name> equipment used in this guide is as follows:
+
+-  Vendor — <vendor-name>
+-  Model — <model name>
+-  Software release — <full software release name>
+
+## Before you begin
+
+1. Review information about how [dynamic routing](https://cloud.google.com/vpn/docs/concepts/choosing-networks-routing#dynamic-routing) works in GCP.
```

Comment:
Our convention is to spell out the first instance of GCP as "Google Cloud Platform."